extensions/sql-database-projects/src/dialogs/deployDatabaseDialog.ts

```diff
@@ -338,6 +384,64 @@ export class DeployDatabaseDialog {
 		return clearButton;
 	}
 
+	private createLoadProfileButton(view: azdata.ModelView): azdata.ButtonComponent {
+		let loadProfileButton: azdata.ButtonComponent = view.modelBuilder.button().withProperties({
+			label: constants.loadProfileButtonText,
+			title: constants.loadProfileButtonText,
+			ariaLabel: constants.loadProfileButtonText,
+			width: '120px'
+		}).component();
+
+		loadProfileButton.onDidClick(async () => {
+			const fileUris = await this.apiWrapper.showOpenDialog(
+				{
+					canSelectFiles: true,
+					canSelectFolders: false,
+					canSelectMany: false,
+					defaultUri: vscode.Uri.parse(this.project.projectFolderPath),
+					filters: {
+						[constants.publishSettingsFiles]: ['publish.xml']
+					}
+				}
+			);
+
+			if (!fileUris || fileUris.length === 0) {
+				return;
+			}
+
+			if (this.readPublishProfile) {
+				const result = await this.readPublishProfile(fileUris[0]);
+				(<azdata.InputBoxComponent>this.targetDatabaseTextBox).value = result.databaseName;
```

Comment:
sible that no db name is there? and in this that case can we keep showing the proj name?

Reply:
Yup in that case, the name will become empty since that's what SSDT does.